version/version.go

```diff
@@ -0,0 +1,7 @@
+package version
+
+var (
+	Version   = "latest"
```

Comment:
Is there actually any chance that this default will be ending up?  If so, would it make sense to have "undefined" or "unknown"?

Reply:
Yes, resolved. Thanks.